blueocean-commons/pom.xml

```diff
@@ -20,8 +20,8 @@
       <artifactId>jackson2-api</artifactId>
     </dependency>
       <dependency>
-          <groupId>org.apache.commons</groupId>
-          <artifactId>commons-lang3</artifactId>
+          <groupId>io.jenkins.plugins</groupId>
+          <artifactId>commons-lang3-api</artifactId>
```

Comment:
as we really don't want it maybe add an enforcer rule to ban `org.apache.commons:commons-lang3`